main.css

```diff
@@ -0,0 +1,743 @@
+    /* Sit on top of the screen */
+    z-index: 1;
+    /* Add a z-index if needed */
+    left: 50%;
+    /* Center the snackbar */
+    bottom: 30px;
+    /* 30px from the bottom */
+}
+
+
+
+#snackbar.show {
+    visibility: visible;
+    -webkit-animation: fadein 0.5s, fadeout 0.5s 2.5s;
+    animation: fadein 0.5s, fadeout 0.5s 2.5s;
+}
+
+#snackbar-warn.show {
+    visibility: visible;
+    -webkit-animation: fadein 0.5s, fadeout 0.5s 2.5s;
+    animation: fadein 0.5s, fadeout 0.5s 2.5s;
+}
+
+.responsive {
+    width: 100%;
+    max-width: 400px;
+    height: auto;
+}
+
+.heading1 {
+    font-weight: 600;
+    font-size: 36px !important;
+}
+
+.heading2 {
+    font-weight: 500;
+    font-size: 30px;
+}
+
+.heading3 {
+    font-weight: 300;
+    font-size: 24px;
+}
+
+.heading4 {
+    font-weight: 200;
+    font-size: 18px;
+}
+
+.heading5 {
+    font-weight: 100;
+    font-size: 14px;
+}
+
+.heading6 {
+    font-weight: 300;
+    font-size: 12px;
+}
+
+.text-color-blue {
+    color: blue;
+}
+
+.text-color-green {
+    color: green;
+}
+
+.small {
+    font-size: 14px;
+}
+
+.modal {
+    background-color: white;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+    width: 40%;
+    padding: 0.5rem;
+    margin: 1rem 1rem 1rem 18rem;
+    /* margin-left: 10rem; */
+    color: black;
+    border-radius: 0.5rem;
+
+
+
+}
+
+.modal-content {
+    padding: 0.5rem;
+}
+
+.form {
+    padding: 0.4rem;
+    width: 50%;
+    background-color: grey;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+    border-radius: 0.5rem;
+    margin: 1rem 1rem 1rem 14rem;
+
+}
+
+.input-form {
+    border: 2px solid black;
+    height: 1.5rem;
+    border-radius: 0.5rem;
+    text-align: center;
+    background-color: white;
+    color: black;
+    width: 40%;
+
+
+}
+
+.error-message {
+    color: orange;
+}
+
+.notification {
+    margin: 1rem;
+    background-color: #555;
+    color: white;
+    text-decoration: none;
+    padding: 15px 26px;
+    position: relative;
+    display: inline-block;
+    border-radius: 2px;
+}
+
+.notification:hover {
+    background: black;
+}
+
+.badge {
+    position: absolute;
+    top: -10px;
+    right: -10px;
+    padding: 5px 10px;
+    border-radius: 50%;
+    background-color: grey;
+    color: white;
+}
+
+.avatar-badge {
+    margin: 1rem;
+    /* background-color: #555; */
+    color: white;
+    text-decoration: none;
+    padding: 15px 26px;
+    position: relative;
+    display: inline-block;
+    border-radius: 2px;
+}
+
+.badgeAvatar {
+    position: absolute;
+    top: 20px;
+    right: 40px;
+    padding: 5px 10px;
+    border-radius: 50%;
+    background-color: yellow;
+    color: black
+}
+
+.Card-with-Shadow {
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+    /* margin-left: 25rem;
+     */
+    margin-inline-start: 25rem;
+
+
+    padding: 0.3rem;
+    border: 1px solid black;
+    width: 9rem;
```

Comment:
Hey!
The image is overlapping on the border, so you can try increasing the `width: 9.5rem`.